osm2gtfs/tests/creators/creators_tests.py

```diff
@@ -0,0 +1,368 @@
+                        "departure_time": row['departure_time'],
+                    }
+                    stop_times1.append(st)
+        with zf2.open("stop_times.txt") as st_file:
+            reader = csv.DictReader(st_file)
+            for row in reader:
+                if row["trip_id"] in trips_id2:
+                    st = {
+                        "trip_id": row['trip_id'],
+                        "stop_id": row['stop_id'],
+                        "stop_sequence": row['stop_sequence'],
+                        "arrival_time": row['arrival_time'],
+                        "departure_time": row['departure_time'],
+                    }
+                    stop_times2.append(st)
+        if len(stop_times1) != len(stop_times2):
+            print("Error on count of stop_times found ({:d} <> {:d})".format(
+                len(stop_times1),
+                len(stop_times2)
+            ))
+            return False
+        # Checking for the first error in stop_times with an explicit message
+        for st1 in stop_times1:
+            trip_point_found = False
+            for st2 in stop_times2:
+                if st1["trip_id"] == st2["trip_id"] \
+                        and st1["stop_sequence"] == st2["stop_sequence"]:
+                    trip_point_found = True
+                    if st1["stop_id"] != st2["stop_id"]:
+                        print(
+                            "stop_id different for trip_id={} & stop_sequence={} ({} <> {})".format(
+                                st1["trip_id"],
+                                st1["stop_sequence"],
+                                st1["stop_id"],
+                                st2["stop_id"]
+                            )
+                        )
+                        return False
+                    elif st1["arrival_time"] != st2["arrival_time"] \
+                            or st1["departure_time"] != st2["departure_time"]:
+                        print(
+                            "Stop times are different for trip_id={} and stop_sequence={}".format(
+                                st1["trip_id"],
+                                st1["stop_sequence"]
+                            )
+                        )
+                        for st in [st1, st2]:
+                            print(
+                                "\t arrival_time={}   departure_time={}".format(
+                                    st["arrival_time"],
+                                    st["departure_time"],
+                                )
+                            )
+                        return False
+                    else:
+                        # trip_point found with no difference, break for st2 loop
+                        # to continue for the next st1
+                        break
+            if not trip_point_found:
+                print(
+                    "No corresponing stop_time for trip_id={} and stop_sequence={}".format(
```

Comment:
*corresponding.
Maybe `matching`

Reply:
There has been no change on this one in this PR. But I'm happy to improve the wording as suggested.